src/key.rs

```diff
@@ -200,6 +200,23 @@ impl DcKey for SignedSecretKey {
     }
 }
 
+/// Deltachat extension trait for secret keys.
+///
+/// Provides some convenience wrappers only applicable to [SignedSecretKey].
+pub trait DcSecKey {
```

Comment:
can we call this `DcSecretKey` please, makes it a bit easier to read in combination with the other names